### ISSUE TITLE
Additional inputs and outputs for method spec

### ISSUE THREAD

**Issue by vkotkas** [CoCoViLa/CoCoViLa#13]
In java methods can use global variables. Usage of these is not presented in the cocovila specifications.

I propose that we should also present the usage of these variables by introducing them in the specification.

An example:

public class Demo {

	double slot = 0;

}

**Comment by MastaP** [comment]
No. Specification variables and Java class fields are completely unrelated, and should be like that. Axiom reflects the type of its realization, this approach breaks it. Also, it does not bring any benefits. Planning algorithm would not work since it's unable to check if class fields are "computable".

**Comment by vkotkas** [comment]
Ok. I see there is room for discussion here. I do not see why class fields are unrelated, why this approach break axiom realization and why planning algorithm will not work.
